### ISSUE TITLE
Many items outdated, list needs to be pruned heavily

### ISSUE THREAD

**Issue by WhatFreshHellIsThis** [xgrommx/awesome-redux#129]
As per title

**Comment by xgrommx** [comment]
yes I know, I'll do when I will have many free time. Sorry, now I have too much tasks in my job.

**Comment by braco** [comment]
@WhatFreshHellIsThis, @xgrommx What items?

**Comment by ryanolsonx** [comment]
emberredux no longer exists

**Comment by gaearon** [comment]
If you’re happy with me doing it, I can walk through all of these and remove the outdated repos as well as those that I feel are not as high quality as others.

**Comment by WhatFreshHellIsThis** [comment]
I can think of no one better qualified @gaearon that would be very helpful thank you.

**Comment by xgrommx** [comment]
@WhatFreshHellIsThis @gaearon Please do =)
